assets/swagger.json

```
@@ -1668,7 +1677,7 @@
         "tags": [
           "ProjectService"
         ],
-        "summary": "Delete a new project token.",
+        "summary": "Delete a new project token",
```

It feels like this should be either:
```suggestion
        "summary": "Delete a project token",
```